cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -64,18 +66,21 @@ void MapBuilderBridge::WriteAssets(const string& stem) {
     LOG(WARNING) << "No data was collected and no assets will be written.";
   } else {
     LOG(INFO) << "Writing assets with stem '" << stem << "'...";
-    if (options_.map_builder_options.use_trajectory_builder_2d()) {
+    CHECK_EQ(trajectory_options_.count(0), 1);
```

Comment:
Please for now add a TODO to support multi trajectory. I opened #343 to discuss what we want to do here.